### PR TITLE
Update retry logic

### PR DESCRIPTION
**Pull request by HemanthDogiparthi12**

### CODE REVIEW

**Review by charliecon — APPROVED** [review]

File: resources/sdk/webmessagingjava/templates/ApiClient.mustache
Comment (dginty4) [inline resources/sdk/webmessagingjava/templates/ApiClient.mustache:1]: Is the web messaging sdk still using our pipeline? I haven't heard anything from the web messaging team in a long time now

File: resources/sdk/purecloudkotlin/templates/ApiClient.kt.mustache
Comment (dginty4) [inline resources/sdk/purecloudkotlin/templates/ApiClient.kt.mustache:1]: Has the kotlin sdk not been removed?

**Review by dginty4 — APPROVED** [review]

File: resources/sdk/purecloudkotlin/templates/ApiClient.kt.mustache
Comment (HemanthDogiparthi12) [reply resources/sdk/purecloudkotlin/templates/ApiClient.kt.mustache:1]: i think it is, added it for consistency

File: resources/sdk/webmessagingjava/templates/ApiClient.mustache
Comment (HemanthDogiparthi12) [reply resources/sdk/webmessagingjava/templates/ApiClient.mustache:1]: believe they are still using it

**Pull request merged**